scripts/dts/edtlib.py

```diff
@@ -1683,9 +1693,11 @@ def _check(self, require_compatible, require_description):
 
         for bus_key in "bus", "on-bus":
```

Comment:
Do you really need `on-bus` to be able to be a list of strings too? The commit log only mentions `bus`, and the test cases don't show how this should work for `on-bus`.

Reply:
I didn't intend for `on-bus` to become a list of strings, just `bus`. I just can't think of a use case where a device would need to have multiple `on-bus`.

Reply:
> I didn't intend for `on-bus` to become a list of strings, just `bus`. I just can't think of a use case where a device would need to have multiple `on-bus`.

OK, so you will update this then, right?

Reply:
Ah, yes... sorry, I didn't understand what you were trying to tell me earlier. Unfortunately, GitHub didn't show the full context of the line you marked so I got confused 😖 . I've split for loop up into 2 if statements now. One for "on-bus", and another for "bus".

Reply:
@mbolivar Is this resolved?